sections/components.xml

```diff
@@ -200,22 +200,52 @@ in the section entitled "GNU Free Documentation License".
 
     <para><link linkend="com-namespaces"><emphasis
-    role="bold">Namespaces</emphasis></link>. All native Joomla 4 component
+    role="bold">Namespaces</emphasis></link>. All native Joomla 4 components
     use <link
     xlink:href="https://www.php.net/manual/en/language.namespaces.php">PHP
-    namespaces</link> for their PHP code. This is a very simple change in
-    thinking for models, views and controllers. For example, instead of your
+    namespaces</link> for their PHP code. This is a simple change in naming
+    classes for models, views and controllers. For example, instead of your
     frontend model being called <code>ExampleModelItem</code> it will now be
     the class <code>ItemModel</code> in the namespace
-    <classname>\<replaceable>My</replaceable>\<replaceable>Prefix</replaceable>\Site\Model</classname>
-    where <code>\My\Prefix</code> is something you can define, and has no
-    restrictions other than it needs to be a valid PHP namespace, unique to
-    your component and should not start with <code>\Joomla</code> (component
-    namespaces starting withIn <code>\Joomla</code> are reserved for core
-    components). This also applies to custom form fields you may need to
-    create in your component — and there <emphasis>is</emphasis> a way to tell
-    Joomla which namespace to look custom form fields in with new attributes
-    in XML forms.</para>
+    <classname>\<replaceable>MyComponentNamespace</replaceable>\Site\Model</classname>
+    where <code>\MyComponentNamespace</code> consists of 3 parts:</para>
+
+    <programlisting>\&lt;vendor part&gt;\Component\&lt;component name&gt;</programlisting>
+
+    <itemizedlist>
+      <listitem>
+        <para><code>\&lt;vendor part&gt;</code>: a valid PHP namespace, unique
+        to you, your company or your project. It should not start with
+        <code>\Joomla</code>, because component namespaces starting withIn
+        <code>\Joomla</code> are reserved for core components. And it should
+        not contain <code>\Component</code>, as that would interfere with the
+        literal part <code>\Component</code> in the namespace. A valid vendor
+        part can for instance be <code>\Acme</code> of
+        <code>\Acme\Support</code>.</para>
+      </listitem>
+
+      <listitem>
+        <para><code>\Component</code>: literally "\Component", with a capital
+        C. </para>
+      </listitem>
+
+      <listitem>
+        <para><code>\&lt;component name&gt;</code>: a valid component name for
+        Joomla as is used in com_componentname. In the namespace-part you can
+        use camel case; in com_componentname only lowercase is used. A valid
+        component name in the namespace would for instance be
+        <code>\Example</code> for component <code>com_example</code>.</para>
+      </listitem>
+    </itemizedlist>
+
+    <para>The namespace of your custom component is also used for custom form
```

Comment:
Technically, it can be used for many different things. Can you please rephrase to something along the lines of “The namespace of your component can also be used elsewhere where a class which belongs to your component might be used. For example, in custom form fields you may…”